test/color_test.dart

```diff
@@ -8,54 +8,53 @@ import 'package:fff/color.dart';
 
 main() {
   test("Constructor without arguments", () {
-    expect(new Color().toString(), "rgb(0, 0, 0)");
+    expect(Color().toString(), "rgb(0, 0, 0)");
   });
 
   test("Constructor with arguments", () {
-    expect(new Color(255).toString(), "rgb(255, 0, 0)");
-    expect(new Color(0, 255).toString(), "rgb(0, 255, 0)");
-    expect(new Color(0, 0, 255).toString(), "rgb(0, 0, 255)");
-    expect(new Color(0, 0, 0, .3).toString(), "rgba(0, 0, 0, 0.3)");
-    expect(new Color(1, 2, 3, 0.4).toString(), "rgba(1, 2, 3, 0.4)");
+    expect(Color(255).toString(), "rgb(255, 0, 0)");
+    expect(Color(0, 255).toString(), "rgb(0, 255, 0)");
+    expect(Color(0, 0, 255).toString(), "rgb(0, 0, 255)");
+    expect(Color(0, 0, 0, .3).toString(), "rgba(0, 0, 0, 0.3)");
+    expect(Color(1, 2, 3, 0.4).toString(), "rgba(1, 2, 3, 0.4)");
   });
 
   test("Output format", () {
-    expect(new Color(255).toString(), "rgb(255, 0, 0)");
-    expect(new Color(255).toRgbString(), "rgb(255, 0, 0)");
+    expect(Color(255).toString(), "rgb(255, 0, 0)");
+    expect(Color(255).toRgbString(), "rgb(255, 0, 0)");
 
-    expect(new Color(255, 0, 0, 1.0).toString(), "rgba(255, 0, 0, 1.0)");
-    expect(new Color(255, 0, 0, 1.0).toRgbaString(), "rgba(255, 0, 0, 1.0)");
+    expect(Color(255, 0, 0, 1.0).toString(), "rgba(255, 0, 0, 1.0)");
+    expect(Color(255, 0, 0, 1.0).toRgbaString(), "rgba(255, 0, 0, 1.0)");
 
-    expect(new Color(255, 0, 0).toHexString(), "ff0000");
+    expect(Color(255, 0, 0).toHexString(), "ff0000");
 
-    expect(new Color(255, 255, 255).toString('red: %r, green: %g, blue: %B'),
+    expect(Color(255, 255, 255).toString('red: %r, green: %g, blue: %B'),
         'red: 255, green: 255, blue: 255');
 
     expect(
-        new Color(255, 255, 255, .5)
+        Color(255, 255, 255, .5)
             .toString('red: %r, green: %g, blue: %B, alpha: %A'),
         'red: 255, green: 255, blue: 255, alpha: 0.5');
 
-    expect(new Color(255, 255, 255, .5).toHexString('#%R%g%b'), '#FFffff');
+    expect(Color(255, 255, 255, .5).toHexString('#%R%g%b'), '#FFffff');
 
     // === toList ===
 
     // simple
-    expect(new Color(0, 0, 0).toList(), equals([0, 0, 0]));
-    expect(new Color(255, 255, 255).toList(), equals([255, 255, 255]));
-    expect(new Color(0, 0, 0, 0.0).toList(), equals([0, 0, 0, 0.0]));
-    expect(
-        new Color(255, 255, 255, 1.0).toList(), equals([255, 255, 255, 1.0]));
+    expect(Color(0, 0, 0).toList(), equals([0, 0, 0]));
+    expect(Color(255, 255, 255).toList(), equals([255, 255, 255]));
+    expect(Color(0, 0, 0, 0.0).toList(), equals([0, 0, 0, 0.0]));
+    expect(Color(255, 255, 255, 1.0).toList(), equals([255, 255, 255, 1.0]));
 
     // as double
-    expect(new Color(10, 20, 30).toList(asDouble: true),
-        equals([10.0, 20.0, 30.0]));
-    expect(new Color(10, 20, 30, 0.4).toList(asDouble: true),
+    expect(
+        Color(10, 20, 30).toList(asDouble: true), equals([10.0, 20.0, 30.0]));
```

Comment:
Could you put this line above to place it immediately after `expect(`?

Reply:
Done.